config/base/300-operator_v1alpha1_addon_crd.yaml

```diff
@@ -48,83 +48,4 @@ spec:
       openAPIV3Schema:
         type: object
         description: Schema for the tektonaddons API
-        properties:
```

Comment:
Any reason for removing this schema ?

Reply:
(Same for the other CRDs)

Reply:
if we want to add schema then we would have to add for all field and it is used for of validation of fields too.  if we keep `x-kubernetes-preserve-unknown-fields: true` allowing all fields from here and webhook would validate and set default similar to pipelines seems to be better.

Reply:
yeah this does means kubectl explain would not show any info 😄
 should we keep the schema or have docs take care of field explanation?

Reply:
@sm43 ideally we would make this auto-generated (it should be possible). Also, those schema might be useful for OpenShift (for auto-creating forms from the CRD when you want to create a new one).

Reply:
@vdemeester currently there isn't a supported way to auto generate crds in knative/pkg I think.
we do it manually, I have created an issue for this https://github.com/tektoncd/operator/issues/352
will address it separately if that's okay ? 🙂

Reply:
Thanks for creating the issue. We tried something on `tektoncd/pipeline` (with annotation and stuff), so it is definitely possible but on `tektoncd/pipeline` we are running out of place for them, here it sohuld work fine, I'll try to link the experiment to the issue 😉